src/svg/ds6/icons.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg aria-hidden="true" hidden>
```

Comment:
Don't think we need this change for an external SVG file. It never gets included as part of the page as far as I'm aware.

Reply:
Oh, my bad. That's copy pasta from the `index.html` file. Thanks for catching that.